### ISSUE TITLE
[BUG] commands like `ssh user@portal 'env'` dont work

### ISSUE THREAD

**Issue by Merith-TK** [moul/sshportal#152]
**What happened**:
i am working on a fork of https://github.com/cdr/sshcode
My fork is adding windows support
When i am at work i use sshportal to bypass the DeepPacket inspection firewall at work (while on break) by using it on termux and running a vpn on my phone

```
$ ./sshcode.exe --skipsync --ssh-flags "-p 2222" codemerith@10.80.38.18
2019-11-26 10:59:42 INFO        OS is windows, disabling connection reuse feature
2019-11-26 10:59:42 INFO        ensuring code-server is updated...
2019-11-26 10:59:44 FATAL       error: failed to update code-server:
---ssh cmd---
ssh -p 2222 codemerith@10.80.38.18 '/usr/bin/env bash -l'
---download script---
set -euxo pipefail || exit 1

[ "$(uname -m)" != "x86_64" ] && echo "Unsupported server architecture $(uname -m). code-server only has releases for x86_64 systems." && exit 1
pkill -f ~/.cache/sshcode/sshcode-server || true
mkdir -p $HOME/.local/share/code-server ~/.cache/sshcode
cd ~/.cache/sshcode
curlflags="-o latest-linux"
if [ -f latest-linux ]; then
        curlflags="$curlflags -z latest-linux"
fi
curl $curlflags https://codesrv-ci.cdr.sh/latest-linux
[ -f ~/.cache/sshcode/sshcode-server ] && rm ~/.cache/sshcode/sshcode-server
ln latest-linux ~/.cache/sshcode/sshcode-server
chmod +x ~/.cache/sshcode/sshcode-server: exit status 255
```
This is what happens when i try and use the program to debug. 

as you can see it tries to run
`ssh -p 2222 codemerith@10.80.38.18 '/usr/bin/env bash -l'` but it does not work, 

**What you expected to happen**:
The program to atleast beable to talk to the server 

**How to reproduce it (as minimally and precisely as possible)**:
Install Termux on android phone
inside termux install go
install SshPortal from source code (git clone ...)
And configure a remote in sshportal
attempt to use `sshcode` (install by `go get -u go.coder.com/sshcode`) with the remote
`sshcode --skipsync --ssh-flags "-p 2222" user@portal`

**Comment by AlexGluck** [comment]
Fix in #55

**Comment by moul** [comment]
Hi @Merith-TK 

Should be fixed with #183 (thx @AlexGluck)

Btw, I like your story and I'm very happy to know that sshportal is used to bypass DPI \o/
